words_discoverer_chrome/background.js

```diff
@@ -466,9 +466,19 @@ function initialize_extension() {
                 useColor: true,
                 color: "blue"
             };
+            user_word_hl_params = {
+                enabled: true,
```

Comment:
My understing was that the list should be disabled by default (until user uploads it)